desktop/docker/windows/Dockerfile

```diff
@@ -49,9 +49,10 @@ RUN mkdir -p $LEIN_INSTALL \
 
 ARG NVM_VERSION
 ARG NODE_VERSION
-RUN curl -s -o- https://raw.githubusercontent.com/creationix/nvm/v${NVM_VERSION}/install.sh | bash
+ADD scripts/3rd-party/nvm/${NVM_VERSION}/install.sh /tmp/nvm/${NVM_VERSION}
```

Comment:
I really don't think that's going to work. Dockerfile paths are relative to the location of the Dockerfile and I don't think we can include a file from outside of that folder.
Maybe we can symlink the script inside of the directory?

Reply:
Oh, that's right! :man_facepalming: I should have tested it before pushing the change...

Reply:
Added symlink

Reply:
Why do you do the whole `scripts/3rd-party/nvm/${NVM_VERSION}/` path? We don't intend to install multiple versions, so why don't you just symlink the script itself in the directory?

Reply:
This is to continue having the version selection being done in the `Dockerfile`, and also to avoid having to regenerate the symlink every time we update a tool version. Makes sense?

Reply:
Oh, I see, you symlinked the whole `3rd-party` folder so the selection would work. I guess that's fine.

Did you check that this works?

Reply:
I ended up switching to an alternative solution, I'm copying the `scripts/3rd-party` tree to the docker context just for the duration of the docker build.